spec/models/basket_spec.rb

```diff
@@ -15,7 +15,7 @@
     it { should belong_to(:user) }
   end
 
-  describe 'Build goods from file appload' do
+  describe '#Build_entries_from_file_appload' do
```

Comment:
Here you should name it exactly the same as the method definition

Reply:
🤣 appload !!!